.gitignore

```diff
@@ -14,6 +14,7 @@ test/oqs_test_groups
 # test artifacts
 tmp
 interop.log
+scripts/openssl-ca-no-oqsprovider.cnf
```

Comment:
This also could be avoided by `rm`ing the temp cnf file in/at the end of the test script (or creating it in the tmp folder).

Reply:
Oh, sure. Changed now.